demisto_sdk/commands/doc_reviewer/rn_checker.py

```diff
@@ -0,0 +1,108 @@
+from typing import Dict
+
+import click
+
+
+def print_template_examples():
+    click.secho('\nGeneral Pointers About Release Notes:', fg='bright_cyan')
+    click.echo(' - The release notes need to be in simple language and informative. '
+               'Think about what is the impact on the user and what they should know about this version.')
+    click.echo(' - Command names - should be wrapped with three stars - ***command-name***')
+    click.echo(' - Packs/Integrations/scripts/playbooks and other content '
+               'entities (incident fields, dashboards...) - should be wrapped with two stars - **entity_name**')
+    click.echo(' - Parameters/arguments/functions/outputs names - '
+               'should be wrapped with one star - *entity_name*')
+    click.secho('\nEnhancements examples:', fg='bright_cyan')
+    click.echo('\n - You can now filter an event by attribute data fields.')
+    click.echo('\n - Added support for the *extend-context* argument in the ***ua-parse*** command.')
+    click.echo('\n - Added 2 commands:\n  - ***command-one***\n  - ***command-two***')
+    click.echo('\n - Added the **reporter** and **reporter email** '
+               'labels to incidents that are created by direct messages.')
+    click.echo('\n - Improved implementation of the default value for the *fetch_time* parameter.')
+    click.secho('\n\nBug fixes examples:', fg='bright_cyan')
+    click.echo('\n - Fixed an issue where mirrored investigations contained mismatched user names.')
+    click.echo('\n - Fixed an issue with ***fetch-incidents***, which caused incident duplication.')
+    click.echo('\n - Fixed an issue in which the ***qradar-delete-reference-set-value*** command failed to '
+               'delete reference sets with the "\\" character in their names.')
+    click.secho('\n\nDocker Updates:', fg='bright_cyan')
+    click.echo('\n - Updated the Docker image to: *demisto/python3:3.9.1.15759*.')
+    click.secho('\n\nGeneral Changes:', fg='bright_cyan')
+    click.secho('Note: Use these if the change has no visible impact on the user, '
+                'but please try to refrain from using these if possible!', fg='yellow')
+    click.echo('\n - Maintenance and stability enhancements.')
+    click.echo('\n - Documentation and metadata improvements.')
+    click.echo('\n\nFor additional information see: https://xsoar.pan.dev/docs/documentation/release-notes')
+
+
+class ReleaseNotesChecker:
+    RN_TEMPLATES = {
+        'You can now', 'Added support for', 'Improved implementation', 'Fixed an issue',
+        'Updated the Docker image to', 'Added the', 'Maintenance and stability enhancements.',
+        'Documentation and metadata improvements.'
```

Comment:
i think this way it more readable, WDYT?
```suggestion
        # prefixes
        'Added support for',
        'Added the',
        'Fixed an issue',
        'Improved implementation',
        'Updated the Docker image to',
        'You can now',
        
        # full line
        'Documentation and metadata improvements.',
        'Maintenance and stability enhancements.',
```